truenas_installer/install.py

```diff
@@ -13,19 +15,24 @@
 BOOT_POOL = "boot-pool"
 
 
-async def install(disks, set_pmbr, authentication, post_install, sql, callback):
+async def install(disks: list[Disk], destination_disks: list[str], wipe_disks: list[str], set_pmbr: bool,
+                  authentication: dict | None, post_install: dict | None, sql: str | None, callback: Callable):
     with installation_lock:
         try:
             if not os.path.exists("/etc/hostid"):
                 await run(["zgenhostid"])
 
-            for disk in disks:
+            for disk in destination_disks:
```

Comment:
You've changed it so that you're iterating `destination_disks` but you're passing the `disks` list object into `format_disks`. This is sloppy. The same goes for `for disk in wipe_disks`. If you kept those there for logging purposes, then we should moving the logging into `format_disk` and `wipe_disk`. Also, because we're not dealing with multiple disks, we should rename the functions to `wipe_disks` and `format_disks` accordingly.